examples/mdx-deck/now.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rewrites": [
+    { "source": "/(.*)", "destination": "index.html" }
+  ]
```

Comment:
Instead of changing the example, `mdx-deck` should be added to zero config framework detection.

There are instructions here: https://github.com/zeit/now/blob/master/.github/CONTRIBUTING.md#add-a-new-framework

Reply:
Agree. Closing the PR.